UI/index.html

```diff
@@ -0,0 +1,338 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Profit!</title>
+		<link rel="stylesheet" type="text/css" href="styles.css">
+	</head>
+	<body>
+		<header>
+			<div class="header__logo">
```

Comment:
The most common way to write classes and ids in HTML is snake-case, let's use it (in this case here should be `header-logo` class).

Reply:
just I read about "block__element" way.
this method cannot be used?

Reply:
It can, but why did you decide to use this particular approach?

Reply:
Because I decided this is a enough intuitive way to navigate in my classes. Easier to write for me